bin/web.js

```diff
@@ -120,7 +121,7 @@ myNuts.init()
 
 // Start the HTTP server
 .then(function() {
-    var server = app.listen(process.env.PORT || 5000, function () {
+    var server = app.listen(process.env.VCAP_APP_PORT || process.env.PORT || 5000, function () {
```

Comment:
I agree with @SamyPesse here, `VCAP_APP_PORT` is [deprecated](https://docs.cloudfoundry.org/devguide/deploy-apps/environment-variable.html#VCAP-APP-PORT), it is recommended to use `PORT` for Cloud Foundry.